soundfile.py

```diff
@@ -467,6 +467,52 @@ def blocks(file, blocksize=None, overlap=0, frames=-1, start=0, stop=None,
             yield block
 
 
+class _SoundFileInfo(object):
+    """Information about a SoundFile"""
+    def __init__(self, file, verbose):
+        self.verbose = verbose
+        with SoundFile(file, 'r') as f:
+            self.name = f.name
+            self.samplerate = f.samplerate
+            self.channels = f.channels
+            self.length = len(f)/f.samplerate
```

Comment:
I wouldn't call this "length". For me, this is a "duration".
Calling it "length" is very confusing.